pkgs/games/xonotic/default.nix

```diff
@@ -73,7 +73,9 @@ let
 
     dontStrip = target != "release";
 
-    buildPhase = lib.optionalString withDedicated ''
+    buildPhase = ''
+      (cd ../d0_blind_id && ./configure --prefix="$out" && make -j $NIX_BUILD_CODE -l $NIX_BUILD_CORES)
```

Comment:
Can d0_blind_id be built in a seperate derivation to keep things simple? All of the things you are doing here are usually done automatically by stdenv.

Reply:
Yeah, we could do that. I figured that there are no other known users of d0_blind_id besides xonotic so it wasn't worth the trouble of doing a separate derivation.